src/components/subNavigation/SubNavigationConnected.tsx

```diff
@@ -23,5 +20,4 @@ const mapDispatchToProps = (dispatch: (action: IReduxAction<IReduxActionsPayload
     onClickItem: (itemId) => dispatch(selectSubNavigation(ownProps.id, itemId)),
 });
 
-export const SubNavigationConnected: React.ComponentClass<ISubNavigationProps> =
-    connect(mapStateToProps, mapDispatchToProps, ReduxUtils.mergeProps)(SubNavigation);
+export const SubNavigationConnected = connect(mapStateToProps, mapDispatchToProps, ReduxUtils.mergeProps)(SubNavigation);

```

Comment:
I think you must use the definition for the left side of the assignent otherwise it create an issue with the d.ts @pochretien had to revert some changes I made for the BackdropConnected because of this